bindings/python/crocoddyl/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ TARGET_LINK_BOOST_PYTHON(${PROJECT_NAME}_pywrap)
 # BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS spews conversion warnings from int to long unsigned int.
 # Unfortunately, using literals does not work in a macro. As such, this turns them off for the entire wrapper:
 TARGET_COMPILE_OPTIONS(${PROJECT_NAME}_pywrap PRIVATE "-Wno-conversion")
+# We are also exposing deprecated declarations until they are removed. Ignore the errors for the wrapper:
+TARGET_COMPILE_OPTIONS(${PROJECT_NAME}_pywrap PRIVATE "-Wno-deprecated-declarations")
```

Comment:
I think it is better to add this flag in each file.
Otherwise, it is harder to detect if we haven't effectively update bits in the Python bindings
Do you agree with me?

Reply:
It's quite a lot of them since we already have a lot of warnings from the Residuals / Frame costs that also need to be silenced

Reply:
I've looked into it and the deprecated Frame declarations are not silenced by the pragmas in the respective files. Due to this, I'll leave the general deactivation in for now until we remove the old Frame signatures. It's too noisy otherwise. For the newly deprecated code, I added it in the file properly as well though.

Reply:
You're referring to clang compilation, right?

I am aware that current code is working on gcc only.